sample/fastmcp_server.py

```
@@ -0,0 +1,334 @@
+        # 转换整数类型参数
+        if class_index is not None:
+            class_index = int(class_index)
+        if class_index1 is not None:
+            class_index1 = int(class_index1)
+        if class_index2 is not None:
+            class_index2 = int(class_index2)
+
+        # 转换 changes 参数，支持 JSON 字符串或字典
+        if changes is not None:
+            if isinstance(changes, str):
+                # 如果是 JSON 字符串，解析为字典
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)